indico/modules/events/management/templates/delete_events.html

```diff
@@ -0,0 +1,30 @@
+{% from 'confirmation_dialog.html' import confirmation_dialog %}
+
+{% set ok_text=_("Delete events") %}
+
+{% set confirmation_message %}
+    {% trans -%}
+        Please note that if you delete the events you will lose all the information contained in them.
+        This operation is irreversible!
+    {%- endtrans %}
+{% endset %}
+
+{% if events %}
+    {% call confirmation_dialog('danger', message=confirmation_message, ok_text=ok_text) %}
+        <input type="hidden" name="submitted">
```

Comment:
@florv  added `name="confirmed"` to the submit button so you can use that to check whether the action has been confirmed